mozilla-release/mobile/android/base/java/org/mozilla/gecko/db/BrowserContract.java

```diff
@@ -361,6 +361,9 @@ private Combined() {}
 
         public static final String LOCAL_VISITS_COUNT = "localVisitCount";
         public static final String LOCAL_DATE_LAST_VISITED = "localDateLastVisited";
+
+        public static final String DATE_LAST_VISITED = "date";
+        public static final String VISITS = "visits";
```

Comment:
You should not change this class, please use the relevant columns from HistoryColumns.